plinth/modules/users/urls.py

```diff
@@ -40,7 +40,7 @@
         name='change_password'),
     # Add Django's login/logout urls
     url(r'^accounts/login/$', public(SSOLoginView.as_view()), name='login'),
-    url(r'^accounts/logout/$', SSOLogoutView.as_view(),
+    url(r'^accounts/logout/$', public(SSOLogoutView.as_view()),
```

Comment:
I think this could be non_admin_view instead of public.

Reply:
Thanks, indeed, this works as well, and seems more appropriate. I pushed a change accordingly.